xkcdpassgen.yaml

```diff
@@ -1,15 +1,16 @@
-name: xkcdpassgen
+name: xkcdpass
 version: 1.0
-summary: xkcd password generator snap
+summary: xkcd password generator
```

Comment:
I think there is no need to mention that it's a snap. This will be showed on the ubuntu store, the type of package is not really important to the users.